templates/Dockerfile.tpl

```diff
@@ -0,0 +1,31 @@
+{{ template "boilerplate_hash" }}
+
+# Build the manager binary
+FROM golang:1.14.1 as builder
+
+ARG work_dir=/github.com/aws/aws-service-operator-k8s
```

Comment:
probably should have noticed this in the commit that added the Dockerfile, but I think the work_dir should be `go/src/github.com/aws/aws-service-operator-k8s/services/{{ .ServiceAlias }}`. The `go/src/` prefix is the default `GOPATH` for the golang builder image and we want to set the package path to the specific service controller...